JitsiConference.js

```diff
@@ -72,6 +67,61 @@ function JitsiConference(options) {
 }
 
 /**
+ * Initializes the conference object properties
+ * @param options overrides this.options
+ */
+JitsiConference.prototype._init = function (options) {
+    if(!options)
+        options = {};
+    if(!this.options) {
+        this.options = options;
+    } else {
+        // Override config options
+        var config = options.config || {};
+        for(var key in config)
+            this.options.config[key] = config[key] || this.options.config[key];
+    }
```

Comment:
I find the above source code implements complex logic that's unjustified (at least I don't see a reason or an explanation about its complexity):
- If there's no this.options, then we're remembering the whole options object.
- But if there's this.options, we care only about the config property of the options object.

Wouldn't it be easier to read & maintain if we paid attention to the whole options object at all times.

Another reason for adopting ES6 is that the above source code of 10 lines could easily be replaced with 1 line:
this.options = Object.assign(this.options || {}, options);